key_commands.rst

```diff
@@ -59,16 +59,15 @@ The output will look as it follows:
 
    Private key listing (/home/joana/.singularity/sypgp/pgp-secret):
 
-   0) U: Johnny Cash (none) <cash@sylabs.io>
-   C: 2019-04-11 22:22:28 +0200 CEST
-   F: 47282BDC661F58FA4BEBEF47CA576CBD8EF1A2B4
-   L: 3072
-   --------
-   1) U: John Green (none) <john@sylabs.io>
-   C: 2019-04-11 13:08:45 +0200 CEST
-   F: 5720799FE7B048CF36FAB8445EE1E2BD7B6342C5
-   L: 1024
-   --------
+   0)  User:              Johnny Cash (none) <cash@sylabs.io>
+       Creation time:     2019-04-11 22:22:28 +0200 CEST
+       Fingerprint:       47282BDC661F58FA4BEBEF47CA576CBD8EF1A2B4
+       Length (in bits):  3072
+
+   1)  User:              John Green (none) <john@sylabs.io>
```

Comment:
I hadn't noticed we had potentially real email addresses in these docs before. While we are here, the email addresses should be changed so they can't possible be real... i.e. `xxxx@example.com`